newsroom/push.py

```diff
@@ -765,7 +764,9 @@ def set_item_reference(coverage):
 def locked(_id: str, service: str):
     lock_name = f"notify-{service}-{_id}"
     if not lock(lock_name, expire=300):
-        raise LockedError(lock_name)
+        logger.debug(f"Lock conflict on {lock_name}")
```

Comment:
I think raising an exception was intentional, maybe to get the exception/stack trace pushed to sentry for further investigation, @petrjasek?

Reply:
Scratch that, I just noticed the exception catch for this type